style.css

```diff
@@ -1,391 +1,568 @@
+    display: grid;
+    grid-template-columns: repeat(3, 0.8fr);
+    gap: 0.5em;
+    min-width: 80%;
+    margin-left: 1em;
+    margin-right: 1em;
   }
 
-  .header-container {
-    margin-top: 4em;
+  .all-skills * {
+    width: auto;
   }
+}
 
-  .code-skill {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
-    width: 80%;
+@media only screen and (min-width: 700px) {
+  #home-paragraph {
+    width: 53.65%;
   }
 
-  .lang {
-    align-self: center;
-    font-size: small;
-    margin-left: 0;
-    margin-right: 5px;
-    margin-bottom: 10px;
-    padding-left: 2px;
-    padding-right: 2px;
-    padding-top: 5px;
-    padding-bottom: 5px;
-    border-radius: 8px;
-    background-color: #fff;
-    color: #36b37e;
+  .aside-container {
+    background: url('images/illustration-about-me-2-desktop.svg') no-repeat, url('images/illustration-about-me-desktop.svg') no-repeat;
+    background-position: left bottom, right;
   }
 
-  #code-lang-list {
-    justify-content: center;
-    min-width: 100%;
+  .header-container-about {
+    margin-top: 1em;
+    background: none;
   }
 
-  #frame-wrk-list {
-    min-width: 70%;
+  .about-paragraph {
+    width: 53.65%;
   }
 
-  .extra {
-    min-width: 100%;
-    padding: 0;
+  .form-sec {
+    max-width: 80%;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: 15em;
   }
 
   .last-head-container {
-    text-align: center;
-    display: flex;
-    justify-content: center;
+    text-align: initial;
+    width: 50%;
+    padding-right: 2em;
+    justify-content: stretch;
+    align-items: stretch;
   }
 
-  .header-3 {
-    font-size: 40px;
-    color: #172b4d;
-    margin-left: 0.5em;
-    margin-right: 0.5em;
+  .last-head-container > * {
+    font-size: 30px;
   }
 
   .form {
-    display: flex;
-    flex-direction: column;
-    margin: 1em;
+    width: 50%;
+    margin-right: 0;
+    margin-top: 3em;
+    border: 2px;
   }
 
-  #name {
+  .names {
     display: flex;
-    flex-direction: column;
-    margin-bottom: 1em;
+    margin-bottom: 0.5em;
   }
 
-  input {
-    padding: 0.7em;
+  .names > input:nth-of-type(1) {
+    margin-right: 0.5em;
   }
 
   #get-in-touch {
-    margin-left: 40%;
-    margin-top: 1em;
+    margin-left: 0;
   }
 
-  .footer * {
-    justify-content: center;
+  .skillset {
+    background:
+      url('images/box.svg') no-repeat,
+      url('images/illustration-2-contact-form-desktop.svg ') no-repeat,
+      url('images/contact-form-illustratoin-3-desktop.svg') no-repeat;
+    background-position: bottom left, 20% 100%, right 100%;
   }
+}
 
-  #code-lang-list * {
-    min-width: 20%;
+@media only screen and (min-width: 900px) {
+  body {
+    background: url('images/Header-llustration-desktop.svg');
+    background-repeat: no-repeat;
+    background-size: 1200px;
   }
 
-  .mail-container {
-    display: flex;
+  .cards {
+    width: fit-content;
+    grid-template-columns: repeat(3, 1fr);
+    grid-column-gap: 1em;
```

Comment:
- [ ] Your use of CSS grid to lay out the cards in three columns on desktop shows your mastery. However, they are not aligned at the center of the page as in the template. Here is a screenshot.

<img width="822" alt="cards" src="https://user-images.githubusercontent.com/71434803/126881101-c1a56182-e519-4dd8-8651-3e03a939f746.png">

The `width: fit-content` property causes the grid to use whatever space it finds, in this case restricting the cards to the side. Kindly remove the property so the cards can be aligned with the rest of your page's content.